orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/util/ArtifactResolver.java

```diff
@@ -163,7 +163,17 @@ public void resolveArtifacts(@Nonnull Map pipeline) {
     List<Artifact> artifactsFromTrigger = (List<Artifact>) Optional.ofNullable((List) trigger.get("artifacts"))
       .map(list -> list.stream().map(it -> objectMapper.convertValue(it, Artifact.class)).collect(toList()))
       .orElse(emptyList());
-    List<Artifact> receivedArtifacts = Stream.concat(receivedArtifactsFromPipeline.stream(), artifactsFromTrigger.stream()).collect(toList());
+
+    List<Artifact> artifactsFromTriggerBuildInfo = emptyList();
+    if (trigger.get("type").equals("manual") && trigger.get("buildInfo") != null) {
```

Comment:
When are we populating the artifacts from build info when it's not a manual execution?

Reply:
It happens in Echo [in here](https://github.com/spinnaker/echo/blob/0718fddd458b14b752acaeaf2795775656f6b60c/echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/monitor/BuildEventMonitor.java#L88)